cmd/neofs-node/config.go

```diff
@@ -418,21 +417,22 @@ type cfg struct {
 	cfgObject         cfgObject
 }
 
-// ReadCurrentNetMap reads network map which has been cached at the
-// latest epoch. Returns an error if value has not been cached yet.
+// GetNetworkMap reads network map which has been cached at the latest epoch.
+// Returns an error if value has not been cached yet.
 //
 // Provides interface for NetmapService server.
-func (c *cfg) ReadCurrentNetMap(msg *netmapV2.NetMap) error {
+func (c *cfg) GetNetworkMap() (netmap.NetMap, error) {
 	val := c.netMap.Load()
 	if val == nil {
-		return errors.New("missing local network map")
+		return netmap.NetMap{}, errors.New("missing local network map")
 	}
 
-	val.(netmap.NetMap).WriteToV2(msg)
-
-	return nil
+	return val.(netmap.NetMap), nil
 }
 
+// CurrentEpoch returns the latest cached epoch.
+func (c *cfg) CurrentEpoch() uint64 { return c.networkState.CurrentEpoch() }
```

Comment:
why `GetNetworkMap` but `CurrentEpoch`?

Reply:
this is a simple getter while `GetNetworkMap` can fail. I didnt think about this much tbh